doc/manual/manual.bib

```diff
@@ -11000,4 +11000,11 @@ @article{buit12
 year={2012}
 }
 
-
+@article{guer11,
+  author = 	 {{Guermond, Jean-Luc, Richard Pasquetti, and Bojan Popov}},
```

Comment:
you need to separate individual authors by "and", not a comma, for bibtex format.